src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -12,8 +12,40 @@
  */
 public class FieldXY {
 
+    public void gogoBender(Robot robot, long i) {
```

Comment:
it's funny) but let's change it to more informative name)

Reply:
Noooo, I has fixed)